analysis/wwz/full_run3_run.sh

```diff
@@ -14,13 +14,13 @@
 #################################################################################################################################
 
 #Run at scale (with futures) for JUST 2022
-#time python run_wwz4l.py ../../input_samples/cfgs/wwz_analysis/samples_4lskim_2022_v1.cfg -o y22_wwz_histos_noSyst -x futures -n 200 --hist-list bdt
```

Comment:
Do we need these changes? Unless there is something substantial here that I'm missing, let's just stick with the current master version

Reply:
fixed